gx_suite_definitions/target_exp_validation_harmonized.ipynb

```diff
@@ -0,0 +1,354 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import synapseclient\n",
+    "\n",
+    "import great_expectations as gx\n",
+    "\n",
+    "context = gx.get_context(project_root_dir='../src/agoradatatools/great_expectations')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create Expectation Suite for Target Exp Validation Harmonized Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Get Example Data File"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "syn = synapseclient.Synapse()\n",
+    "syn.login()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "target_exp_validation_harmonized_data_file = syn.get(\"syn25740978\").path\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create Validator Object on Data File"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "validator = context.sources.pandas_default.read_json(\n",
+    "    target_exp_validation_harmonized_data_file\n",
+    ")\n",
+    "validator.expectation_suite_name = \"target_exp_validation_harmonized\"\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Add Expectations to Validator Object For Each Column"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ensembl_gene_id\n",
+    "validator.expect_column_values_to_be_of_type(\"ensembl_gene_id\", \"str\")\n",
+    "#validator.expect_column_values_to_not_be_null(\"ensembl_gene_id\")\n",
```

Comment:
There is one row with a`null` `ensembl_gene_id`. Is this a mistake? I left this in, but commented out for the moment.

Reply:
Ahh yeah, this is an old issue that's tracked in https://sagebionetworks.jira.com/browse/AG-905.

I'll sync up with Jaclyn and we'll figure out what to do about it. Hopefully we can solve the issue quickly and not merge with commented out expectations that we have to track...

Reply:
I updated the data file to remove the N/A row. If you pull `dev` into this branch it should pick up the update, and then you should be able to uncomment this validation.

Reply:
OK I just pushed that change